target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -136,12 +136,15 @@ ramips_setup_macs()
 		;;
 	mikrotik,routerboard-750gr3|\
 	mikrotik,routerboard-760igs|\
-	mikrotik,routerboard-m11g|\
 	mikrotik,routerboard-m33g)
 		label_mac=$(cat "/sys/firmware/mikrotik/hard_config/mac_base")
 		wan_mac=$label_mac
 		lan_mac=$(macaddr_add $label_mac 1)
 		;;
+	mikrotik,routerboard-m11g)
+		label_mac=$(cat "/sys/firmware/mikrotik/hard_config/mac_base")
+		lan_mac=$label_mac
```

Comment:
Is changing the assigned address by -1 also correct?

Reply:
This was intentional: I think it is correct. I do not have a device to check. On any of the other Mikrotik devices I have, `/sys/firmware/mikrotik/hard_config/mac_base` is used as first ethernet MAC.

Reply:
"Any other" is no help for MAC addresses. Vendors are rarely consistent with MAC addresses. Please only change the effective MAC address when you have verified it. Separating the device and not setting non-existent wan is fine, but the +1 should stay until verified otherwise.